Prog/clean-montgomery.Rmd

```diff
@@ -538,6 +538,15 @@ jur <- jur %>%
 rm(garden, highrise, other, condo, mixeduse, apartment)           
 ```
 
+
+Overwrite jur with new categorizations, remove intermediate data frames
```

Comment:
i don't want you to overwrite jur...i'm fine if you remove some of the intermediate steps occasionally but the point of renaming them is so one can compare versions.